subworkflows/local/prodigal.nf

```diff
@@ -0,0 +1,28 @@
+//
+// Run prodigal as orf caller then generate nice format for gff
+//
+
+include { PRODIGAL as PRODIGAL_MODULE    } from '../../modules/nf-core/prodigal/main'
+include { FORMAT_PRODIGAL                } from '../../modules/local/format_prodigal.nf'
+include { UNPIGZ as UNPIGZ_FASTA_PROTEIN } from '../../modules/local/unpigz.nf'
```

Comment:
You're not using this, I think.

Reply:
right. I removed it . I noticed that I didn't need to unpigz while I was working on it but I forgot it there.